### PR TITLE
Tom only show export spinner on empty total grids

### PR DESCRIPTION
**Pull request by Derryrover**

### CODE REVIEW

File: src/utils/geoUtils.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+export const getSpatialBoundsIntersect = (bounds1, bounds2) => {
+  const intersectSpatialBounds = {
+    north: bounds1.north < bounds2.north ? bounds1.north : bounds2.north,
+    east: bounds1.east < bounds2.east ? bounds1.east : bounds2.east,
+    south: bounds1.south > bounds2.south ? bounds1.south : bounds2.south,
+    west: bounds1.west > bounds2.west ? bounds1.west : bounds2.west,
+  }
+  if (
+      intersectSpatialBounds.north < intersectSpatialBounds.south ||
+      intersectSpatialBounds.east < intersectSpatialBounds.west
```
Comment (GeOdin) [inline src/utils/geoUtils.ts]: What happens when you make a big square around New Zealand, or the east part of Russia/ west part of Canada in the EPSG:3857 projection?

The west (left) side of the selection becomes 179,.. and the east (right) side of the selection becomes -179,.. .

This is one edge case where the east has a lower value than the west, but the selection is still valid.

This does not have to be corrected now. It is an edge case (literally) that does not happen often. I will make a new ticket.

File: src/catalogue/styles/Export.css [hunk 2]
```diff
@@ -59,6 +59,7 @@
     outline: auto;
     height: 93%;
     display: flex;
+    position: relative;
 }
 
 /* CONTENT */
```
[the remark this answers is not in the document]
Comment (Derryrover) [reply src/catalogue/styles/Export.css]: -removed comment
- changed class to export_rightsidebar

File: src/catalogue/components/ExportModal.tsx [hunk 3]
```diff
@@ -73,46 +77,25 @@ class ExportModal extends React.Component<MyProps> {
...
 
         return (
             <div className="export_main">
                 <div className="export_map-selection">
                     <h3>Export Selection</h3>
-                    <div className="export_map-box" style={{position:"relative"}}>
+                    <div className="export_map-box">
                         { 
-                            this.props.fetchingGridState !== "RECEIVED"?
+                            (fetchingGridState === "SENT" || fetchingGridState === "NOT_SENT") && exportGridCells.length===0 ?
```
[the remark this answers is not in the document]
Comment (Derryrover) [reply src/catalogue/components/ExportModal.tsx]: added comment 
                            // show spinner if gridcells are not yet retrieved

File: src/utils/geoUtils.ts (hunk 1 above)
Comment (Derryrover) [reply src/utils/geoUtils.ts]: I added a comment for Joeri on 
https://nelen-schuurmans.atlassian.net/browse/FRNT-592

**Pull request merged**